### PR TITLE
doc: fix writingchecks.xml docs

### PR DESCRIPTION
**Pull request by nmancus1**
Update documentation to reflect comments at https://github.com/checkstyle/checkstyle/issues/8050#issuecomment-898897547.

Exact link to docs: https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/26458c6_2021135057/writingchecks.html#Limitations

image

### CODE REVIEW

**Comment by nmancus1** [comment]
Github, generate site

**Review by rnveach — APPROVED** [review]
You could add `until Issue #XXXX` and add a note in issue to remove this comment.

File: src/xdocs/writingchecks.xml [hunk 1]
```diff
@@ -665,7 +665,10 @@ public class MethodLimitCheck extends AbstractCheck
           no
           <a href="https://docs.oracle.com/javase/specs/jls/se8/html/jls-3.html#jls-3.3">
             Unicode escape sequence </a>
-          support (i.e. <code>\uXXXX</code>) in keywords and identifiers.
+          support (i.e. <code>\uXXXX</code>) in keywords and identifiers. Javadoc
```
Comment (romani) [inline src/xdocs/writingchecks.xml:668]: Move "Javadoc" to new line. Please add link https://github.com/checkstyle/checkstyle/issues/10629 until what time it is limitation.
It is not permanent limitation, it is unfortunate lack of time, but we should support it .

**Review by romani — CHANGES_REQUESTED** [review]
Items:

File: src/xdocs/writingchecks.xml (hunk 1 above)
Comment (nmancus1) [reply src/xdocs/writingchecks.xml]: Done.

**Comment by nmancus1** [comment]
Github, generate site

**Review by romani — APPROVED** [review]

**Pull request merged**